concurrency/2.0/_index.md

```diff
@@ -1,23 +1,23 @@
 ---
 title: "Jakarta Concurrency 2.0 (under development)"
```

Comment:
Change "(under development)" to "(milestone)" to be clear that you've met the "milestone 1 criteria" and to be consistent with those that came before you...  Thanks!